server/repository/migrations/postgres/2022-02-11T15-00_create_key_value_store/up.sql

```diff
@@ -1,9 +1,14 @@
 -- https://github.com/openmsupply/open-msupply/blob/611f59207bd46178844c99c267f75115a721cc1c/server/repository/src/db_diesel/key_value_store.rs#L21-L40
-CREATE TYPE key_type AS ENUM (
+CREATE TYPE key_type AS ENUM
+(
     -- Cursor for pulling central records from the central server
     'CENTRAL_SYNC_PULL_CURSOR',
     -- Cursor for pushing remote records to central server
     'REMOTE_SYNC_PUSH_CURSOR',
+    -- display settings
+    'SETTINGS_DISPLAY_CUSTOM_THEME',
```

Comment:
I think the new `up` migration you've added should be enough, we wouldn't need this variants added here if they are added with that script, and also wouldn't need to do IF NOT EXISTS